$NetBSD: patch-glib_meson.build,v 1.10 2022/07/24 21:24:56 wiz Exp $

- Don't bother with gtester as it's slated for deprecation anyway:
  https://gitlab.gnome.org/GNOME/glib/issues/1441

--- glib/meson.build.orig	2022-05-30 13:26:47.000000000 +0000
+++ glib/meson.build
@@ -431,25 +431,8 @@ if host_system == 'windows'
       include_directories : configinc,
       dependencies : [libglib_dep])
   endif
-else
-  gtester = executable('gtester', 'gtester.c',
-    install : true,
-    c_args : ['-UG_DISABLE_ASSERT'],
-    include_directories : configinc,
-    dependencies : [libglib_dep])
 endif
 
-report_conf = configuration_data()
-report_conf.set('GLIB_VERSION', glib_version)
-report_conf.set('PYTHON', python_name)
-configure_file(
-  input: 'gtester-report.in',
-  output: 'gtester-report',
-  install_dir: get_option('bindir'),
-  configuration: report_conf,
-  install_mode: 'rwxr-xr-x'
-)
-
 install_data('glib_gdb.py', install_dir : join_paths(glib_pkgdatadir, 'gdb'))
 
 # This is needed to make gdb find glib_gdb.py
